samples/facetime/app/src/main/java/io/dyte/sample/facetime/ControlBarView.kt

```diff
@@ -0,0 +1,52 @@
+package io.dyte.sample.facetime
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.constraintlayout.widget.ConstraintLayout
+import dyte.io.uikit.view.controlbarbuttons.DyteCameraToggleButton
+import dyte.io.uikit.view.controlbarbuttons.DyteLeaveButton
+import dyte.io.uikit.view.controlbarbuttons.DyteMicToggleButton
+import io.dyte.core.DyteMobileClient
+
+class ControlBarView: ConstraintLayout {
```

Comment:
(suggestion): We can extent this class directly from `DyteControlBarView` or directly use that component in MeetingView's XML.